pyspecProcScripts/integrate_limits.py

```diff
@@ -7,32 +7,26 @@
 from matplotlib.pyplot import annotate
 from .apod_matched_filter import apod_matched_filter
 
-logger = init_logging("debug")
-
 def integrate_limits(s, axis="t2",
-        filter_width=100,
         convolve_method='gaussian',
+        cutoff = 0.25,
         fl=None):
-    r"""
-    Integrate Limits
-    ============================
-
-    This function takes data in the frequency
+    r"""This function takes data in the frequency
     domain and finds the corresponding frequency
     limits of the signal for integration.
+
     Limits are found by applying a matched filter
-    (choice of Lorentzian or Gaussian) to the
+    (choice of Lorentzian or Gaussian ``convolve_method``) to the
     frequency domain data, which it then uses to
     determine the integration limits based on a
-    cut off from the maximum signal intensity.
+    ``cutoff`` amplitude based as a faction of the maximum signal intensity.
+
+    Parameters
+    ==========
```

Comment:
hopefully this is self-explanatory

Reply:
I am assuming fraction is what you meant above, so I will change that in a commit